__tests__/app.test.js

```diff
@@ -44,4 +44,29 @@ describe("app", () => {
       return request(app).get("/api/banana").expect(404);
     });
   });
+  describe("/api/articles", () => {
+    it("returns a 200 status : GET responds with an array of article objects with the relevant keys", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          for (let article of body) {
```

Comment:
Include check of the body's length. Also we want our data on a key that suits the type of data we return.